Get-Started.md

```diff
@@ -0,0 +1,111 @@
+# Tutorial
+- This tutorial will guide you from the very basics to real example of IPLD.
+- IPLD stands for the Inter Planetary Linked Data
+- When we get data from the internet, we know it comes from the right place, if the URL is https (some times not even! TODO: add symantec & fake certificates), but we don’t know if it was the data we intended to get!
+- The same problem is with links, we don’t know if a link is still linking to the right page
+- Finally, the general problem of the current web is that we trust the source to give us the right data, so other computers in the network cannot really help sharing the same data!
+- These 3 properties are secure linking, distributed naming and immutable content!
+- Where does IPLD fit into this? IPLD is a data format to describe datasets where data can link to other data via cryptographic hashing, IPLD also provides a path scheme to point to some data in the datasets.
+- Let’s unroll what we just said.
+
+# The Basics
+## Data model
+- Let’s start explaining IPLD and its data model.
+- JSON is one of the most popular semi-structured data formats on the Web.
+- The key reason JSON has great adoption is their simplicity.
+- Although we don’t really “use” JSON in IPLD,  we use the the same notation
+
+```
+object1 = {
+  name: “Mr. Slippery”,
+  job: [“Writer”, “Spy”]
+}
+// Mr. Slippery is the main character of True Names by Vinge
+```
+
+- Alright, this is a JSON object, which is also a valid IPLD object.
+
+### Naming things with hashes
+- In IPLD, we refer to objects with their cryptographic hash.
+- A cryptographic hash is a one way function (link to wikipedia).
+- For any given input it returns a string of bits of the same size.
+- Every object has one cryptographic hash.
+- It is impossible (well, really really difficult) given a hash to know what content it refers to 
+- and given two hashes to know if the content was similar.
+
+- The hash of object1 is XXXX (well, this is a multihash, read more here)
+- if we remove the dot from Mr, the hash is YYYY.
+- In IPLD cryptographic hashes ensure that the content cannot change.
+- If we ask to a network (like the ipfs network) for a hash
+- then, they cannot lie about the content that they are giving
+- since you can generate the hash of the content received and check if it is the same one to the one that you asked for!
```

Comment:
since you can generate the hash of the content received and check if it is the same one as the one that you asked for!